docs/migration_api_keys_to_iam_auth.md

```diff
@@ -0,0 +1,42 @@
+# Migrating from API Key to IAM Auth Access
+
+1. Upgrade to kubectl-hyperkube plugin to 0.6.0 or newer
+
```

Comment:
Would be good to give a reminder of the kubectl plugin installation and upgrade instructions here

Reply:
Good call! adding